src/app/help.rs

```diff
@@ -474,6 +474,7 @@ impl<'a> Help<'a> {
                                            Format::None(pv.to_string_lossy())
                                        }));
             }
+            // FIXME: add [env::{}: {}] here
```

Comment:
to do this, I'll need to keep a reference to the original OsStr key for the env.

Reply:
Since it's meant for display, I'm fine with using `to_string_lossy` to create a displayable string. If it becomes an issue, we can always change it to an OsStr and display the actual bytes later in a non-breaking way.